app/commands/file.py

```diff
@@ -234,12 +233,8 @@ def file_put(**kwargs):  # noqa: C901
         if source_file:
             upload_event['source_id'] = src_file_info
 
-        item_ids = simple_upload(upload_event, num_of_thread=thread, output_path=output_path)
-
-        # since only file upload can attach manifest, take the first file object
-        srv_manifest.attach_manifest(attribute, item_ids[0], zone) if attribute else None
+        _ = simple_upload(upload_event, num_of_thread=thread, output_path=output_path)
```

Comment:
probably we could do?
```suggestion
        simple_upload(upload_event, num_of_thread=thread, output_path=output_path)
```